### ISSUE TITLE
Need to Zoom in WPF WebView

### ISSUE THREAD

**Issue by meshawn** [windows-toolkit/Microsoft.Toolkit.Win32#42]
Feature request (UserVoice request: ) 

## Current behavior
Unable to zoom

## Expected behavior
Able to zoom

## Minimal reproduction of the problem with instructions
I didn't create a user voice request for this because I'm not sure if there is already a way to do this.
I am able to do this on win7/8 with WebViewCompatible by casting/using IE COM apis.  

## Environment

Nuget Package(s): Microsoft.Toolkit.Wpf.UI.Controls.WebView

Package Version(s):  5.0.1

Windows 10 Build Number:
- [ ] Fall Creators Update (16299)
- [ ] April 2018 Update (17134)
- [ ] October 2018 Update (17763)
- [X] Insider Build (build number: (18305.1003)

App min and target version:
- [ ] Fall Creators Update (16299)
- [X] April 2018 Update (17134)
- [ ] October 2018 Update (17763)
- [ ] Insider Build (xxxxx)

Device form factor:
- [X] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

Visual Studio 
- [X] 2017 (version: )
- [ ] 2017 Preview (version: )

```

**Comment by verelpode** [comment]
I agree, need zoom ability in the same manner as in the full MS Edge app.  Presumably the [Windows.Web.UI.Interop.WebViewControl.Scale](https://docs.microsoft.com/en-us/uwp/api/windows.web.ui.interop.webviewcontrol.scale#Windows_Web_UI_Interop_WebViewControl_Scale) property should be used to implement a zoom in/out feature the same as in the full MS Edge app, but unfortunately this `Scale` property exists only in `WebViewControl` and not in `Microsoft.Toolkit.Wpf.UI.Controls.WebView` 6.0.0-preview7.

Considering that WPF WebView uses `WebViewControl` internally, is it possible to make the `WebViewControl.Scale` property available in `Microsoft.Toolkit.Wpf.UI.Controls.WebView` ?

To clarify, a zoom feature should change the size of the webpage/content _inside_ the WebView but not change the size of the WebView itself.

**Comment by atiyka** [comment]
I'd like this feature as well, what is it's state currently?
